API/BackgroundWorkers/OsuMatchDataWorker.cs

```diff
@@ -125,30 +123,11 @@ IGamesRepository gamesRepository
             {
                 if (!ScoreAutomationChecks.PassesAutomationChecks(score))
```

Comment:
```suggestion
                score.IsValid = ScoreAutomationChecks.PassesAutomationChecks(score)
```
If we don't do anything but assign `score.IsValid`, we can just use the bool returned from it instead of using an if.